oshop-dashboard/src/app/app.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.scss']
```

Comment:
this file is empty lets blow this line away and remove the file. Everything we check in should serve a purpose. Never check anything that _might be used_.